README.md

```diff
@@ -29,7 +29,7 @@ For more information about a production deployment, the how-to-guide
 A usual deployment of the charm can be done with the following command (please replace items in `<>` with your own values):
 
 ```bash
-juju deploy github-runner --channel=latest/stable --config openstack-clouds-yaml="$(cat clouds.yaml)" --config openstack-flavor=<flavor> --config openstack-network=<openstack-network> --config path=<org>/<repo> --config token=<github-token>
+juju deploy github-runner --channel=local-lxd/stable --config openstack-clouds-yaml="$(cat clouds.yaml)" --config openstack-flavor=<flavor> --config openstack-network=<openstack-network> --config path=<org>/<repo> --config token=<github-token>
```

Comment:
I think it is fine to use latest/stable here, as we are using openstack runners in this example.

Reply:
The issue is that if we change the main branch, we will not update this README.md file in this branch, and it may stop working. WDYT?

Reply:
You are right (I thought its a merge to main).